find_max.py

```diff
@@ -0,0 +1,16 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index):
+    max = None
+    if max_index == 0:
+        print("Array is empty")
+    else:
+        max = array[0]
+        index = 0
```

Comment:
Maybe set index = 1? That way we avoid an extra comparison in the following logic